packages/vue/src/components/atoms/SfButton/SfButton.vue

```diff
@@ -55,13 +56,24 @@ export default {
       type: String,
       default: "button",
     },
+    type: {
+      type: String,
+      default: null
+    }
   },
   linkActive(link, disabled) {
     return link && disabled;
   },
   buttonActive(link, disabled) {
     return !link && disabled;
   },
+  setType(type, link) {
+    if (type === null && !link) {
+      return 'button'
```

Comment:
would be good to add validation like it was in prop - valid are "button", "submit", "reset"